src/components/PeopleTable.tsx

```diff
@@ -10,635 +107,160 @@ export const PeopleTable = () => {
           <th>
             <span className="is-flex is-flex-wrap-nowrap">
               Name
-              <a href="#/people?sort=name">
+              <SearchLink
+                params={{
+                  sort: 'name',
+                  order: sortOrder === 'asc' ? 'desc' : 'asc',
+                }}
+                onClick={() => handleSort('name')}
```

Comment:
The `onClick` handler for `SearchLink` is redundant since `SearchLink` will automatically handle navigation based on the `params` prop. Consider removing the `onClick` handler to avoid potential conflicts.